### DISCUSSION TITLE
Academic Documents

### DISCUSSION THREAD

**Discussion by Monkeyzuma** [community/community#124846] [New to GitHub]
I want to join Github Education under my college, but I can't. Github wants my academic Documents before I can join, but I cannot find where I am supposed to go to upload the information in my profile.
I am assuming I would have to create a private repository, but I am not sure that is correct.
Can someone help me?

**Comment by davevad93** [comment]
Hi @Monkeyzuma ,

To join GitHub Education and get your student benefits you have to submit your application [here](https://education.github.com/discount_requests/application). If you are a student select `student` in `Select your academic status` and follow the instructions to make your application, you have to put the name of your school and upload a proof of your academic affiliation such as an ID card, or your most recent exam results. The most important thing is that the document has to show apart from your personal data, the name of the educational institution and a date, not only the academic year, but a real date that can prove your actual enrollment otherwise your request could be rejected.
